atom/browser/session_preferences.cc

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) 2017 GitHub, Inc.
+// Use of this source code is governed by the MIT license that can be
+// found in the LICENSE file.
+
+#include "atom/browser/session_preferences.h"
+
+#include "atom/common/options_switches.h"
+#include "base/command_line.h"
+#include "base/memory/ptr_util.h"
+
+namespace atom {
+
+namespace {
+
+#if defined(OS_WIN)
+const base::FilePath::CharType kPathDelimiter = FILE_PATH_LITERAL(';');
+#else
+const base::FilePath::CharType kPathDelimiter = FILE_PATH_LITERAL(':');
+#endif
+
+}  // namespace
+
+// static
+int SessionPreferences::kLocatorKey = 0;
```

Comment:
Would it be better to have the key as `SessionPreferencesUserData` just so it doesn't collide with anything from content layer in the future ?

Reply:
The static member data has the class name in its symbol name, so we won't get symbol conflicts. The `kLocatorKey` is a pattern also used in other Chromium classes.

Reply:
:+1: